core/chainlink.Dockerfile

```diff
@@ -1,10 +1,10 @@
-# Build Chainlink
-FROM smartcontract/builder:1.0.31 as builder
+# MAKE ALL CHANGES WITHIN THE DEFAULT WORKDIR FOR YARN AND GO DEP CACHE HITS
+# THIS LINE IS AUTOGENERATED, DO NOT CHANGE MANUALLY
+FROM smartcontract/cache:0.0.1-1585787725
```

Comment:
It's not clear to me that this is based on the builder, nor what source version of the builder it is using.

Could we use a version that reflects the original builder image tag and perhaps a name like `builder-cache`?

Reply:
Its always using the latest builder image (up to 24h delay), and the cache file is in the repo so Im not sure this is needed. Down to rename to `builder-cache` though.

Reply:
Can we not have it use the latest builder image? It would be better to exclusively use a tagged version, future builder images can often cause breakages and we don't want to break this build.

Reply:
Sorry I should clarify, it uses the latest tagged version. If you look at the linked PR it created on my fork, it pulls the newest tag and includes that in the PR, so you always know which builder version will be included

Reply:
Yeah, I'd like it to not do that, I want to always explicitly have to opt into a new tag, not to use a new one. 

It'd be nice if the `builder-cache` image used a version that reflected the builder image, e.g: `builder-cached:1.30.0-20020206` so that I can see which version it's using.

Reply:
how do you want to include the version of the cache image itself then? We also want to include that so you can manually change the cache file if needed

Reply:
Something like `cache:0.0.1-builder1.30.0-2020-04-06T2004Z`?